crates/test-utils/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "hieratika-test-utils"
+version.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license-file.workspace = true
+
+authors.workspace = true
+description = "The utilities to test Hieratika."
+keywords.workspace = true
+categories.workspace = true
+
+edition.workspace = true
+rust-version.workspace = true
+
+# Make these specifically impossible to publish as they have no independent use.
+publish = false
+
+[dependencies]
+anyhow.workspace = true
+cairo-lang-compiler.workspace = true
+cairo-lang-runner = { path = "../../cairo/crates/cairo-lang-runner" }
+cairo-lang-sierra = { path = "../../cairo/crates/cairo-lang-sierra" }
+cairo-lang-sierra-generator = { path = "../../cairo/crates/cairo-lang-sierra-generator" }
+cairo-lang-utils = { path = "../../cairo/crates/cairo-lang-utils" }
```

Comment:
I feel like these deps may end up used in more than one place, so please hoist them to workspace level and use them here that way. Otherwise we just end up with people duplicating these paths across multiple Cargo projects which is ugly and harder to maintain if something changes.